Altair/admin product

altair201 opened the pull request:
perdon por la tardansa juanpo termine mas tarde de lo que crei, el modal esta recibiendo el product por props solo hace falta que lo mandes al componente para que lo puedas utilizar y de esa forma se vea en el formilario, no se ve bien en el modal por lo que voy a trabajar en eso, por lo que por ahora podes mover con eso mientras ajusto el modal.

altair201 committed:
table

altair201 committed:
table update

altair201 committed:
table modal

altair201 committed:
table modal update

altair201 committed:
table modal update

altair201 committed:
table modal update

altair201 committed:
table update

altair201 committed:
table update

altair201 committed:
Merge branch 'develop' of https://github.com/QuickCodeFusion/Alma-Cannabica into altair/adminProduct

altair201 committed:
table Edit product

altair201 committed:
table Edit product

altair201 committed:
Merge branch 'develop' of https://github.com/QuickCodeFusion/Alma-Cannabica into altair/adminProduct

altair201 committed:
edit product

juanpo12 reviewed (changes requested):
Las categorias se concatenan y parece un mismo string todo aplastado. Aparte de eso no es responsive

altair201 committed:
condicional button

UFCampos committed:
feat[api/carrousel/route] POST now copies product instead of creating a new one

UFCampos committed:
product now has a new attribute when added to carousel

altair201 committed:
fase de prueba no tocar

altair201 committed:
btn ready

UFCampos committed:
refactor product types to follow DRY

UFCampos committed:
refactor[api/carrousel] to not include inCarousel attribute in collection
DELETE Method now also changes inCarousel attribute to false in products collection

UFCampos committed:
Merge branch 'develop' of https://github.com/QuickCodeFusion/Alma-Cannabica into altair/adminProduct

UFCampos committed:
feat[Sidebar]: AdmProducts is now lazy loaded

UFCampos committed:
feat[/products] RTKQuery is now done by page
to avoid useless fetch

altair201 committed:
update ready

altair201 committed:
update ready

juanpo12 commented:
RESOLVE EL CONFLICTO ANTES

UFCampos committed:
Merge branch 'develop' of https://github.com/QuickCodeFusion/Alma-Cannabica into altair/adminProduct

UFCampos committed:
Merge branch 'altair/adminProduct' of https://github.com/QuickCodeFusion/Alma-Cannabica into altair/adminProduct
